kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -208,6 +222,68 @@ apt_get_install()
     wait_for_apt_locks
 }
 
+generate_api_model()
+{
+    touch $DEFINITION_TEMPLATE_NAME
+    cat > $DEFINITION_TEMPLATE_NAME <<EOL
+    {
+    "apiVersion": "vlabs",
+    "location": "",
+    "properties": {
+        "orchestratorProfile": {
+            "orchestratorType": "Kubernetes",
+            "orchestratorRelease": "",
+            "kubernetesConfig": {
+                "kubernetesImageBase": "",
+                "useInstanceMetadata": false,
+                "networkPlugin": "kubenet",
+                "containerRuntime": "docker",
+                "cloudProviderRateLimitBucket": 0,
+                "cloudProviderRateLimitBucketWrite": 0,
```

Comment:
I do not think we want to set `cloudProviderRateLimitBucket` and `cloudProviderRateLimitBucketWrite`.

See https://github.com/Azure/aks-engine/pull/2832